test/k8sT/Updates.go

```diff
@@ -144,6 +138,13 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 		Expect(err).To(BeNil(), "Cilium %q was not able to be deployed", oldVersion)
 		ExpectCiliumReady(kubectl)
 
+		By("Installing kube-dns")
+		_ = kubectl.Apply(helpers.DNSDeployment())
+
+		// Deploy the etcd operator
+		By("Deploying etcd-operator")
+		err = kubectl.DeployETCDOperator()
+		Expect(err).To(BeNil(), "Unable to deploy etcd operator")
 		By("Cilium %q is installed and running", oldVersion)
```

Comment:
Quick one, with that changes that line should be in the top, 140 no? After ExpectCiliumReady I think.

I'm refering to this line: 
```
By("Cilium %q is installed and running", oldVersion)
```